addons/account/static/tests/account_move_form.test.js

```diff
@@ -0,0 +1,54 @@
+import { test } from "@odoo/hoot";
+import { Deferred } from "@odoo/hoot-mock";
+import {
+    assertSteps,
+    click,
+    defineMailModels,
+    insertText,
+    openFormView,
+    start,
+    startServer,
+    triggerHotkey,
+    step,
+} from "@mail/../tests/mail_test_helpers";
+import { defineModels, fields, models, onRpc } from "@web/../tests/web_test_helpers";
+
+class AccountMove extends models.Model {
```

Comment:
I think we should define models in their own folders like we did for mail & other modules we converted.